src/Playground.tsx

```diff
@@ -21,7 +22,7 @@ const CustomPlayground = (props: CustomPlaygroundProps): React.ReactNode => {
   };
 
   return (
-    <ThemeContext.Provider value={{ theme: DefaultTheme, selectedTheme: selectedTheme }}>
+    <UiView theme={DefaultTheme} selectedTheme={selectedTheme}>
```

Comment:
This is the custom playground component that we use in the docs, so by using the `UiView` now all examples are wrapped into the global styles.